README.md

```diff
@@ -17,50 +18,45 @@ The current Swift Server work group consists of the following people:
-* Packages can live in any publicly accessible open source repository and should adopt Apache License v2
+In various situations the SSWG shall hold a vote. These votes can happen on the phone, email, or via a voting service, when appropriate. SSWG members can either respond "agree, yes, +1", "disagree, no, -1", or "abstain". A vote passes with two-thirds vote of votes cast based on the SSWG charter. An abstain vote equals not voting at all.
+
+## Incubation Process
 
-The Swift Server work group will direct the incubation of new server-focused libraries and tools through four phases: Determine Need, Proposal, Prototyping and Development, and Recommendation.
+See [Incubation Process](process/incubation.md)
 
-### Determine Need
-To start this process, the work group first determines and maintains a prioritized list of what’s needed for development and deployment of Swift server applications. The purpose of this list is to seed the community with ideas that will help solicit the creation of proposals.
+## Projects
 
-### Proposal
-As proposals are brought forth by the community, the work group will evaluate them against the following criteria:
+| Project | Sponser | Maturity Level | Pitched | Accepted |
+|---|---|---|---|---|
+| [SwiftNIO](http://github.com/apple/swift-nio/) | Apple | Graduated  | N/A  | 9/7/2018  |
 
-* Does it provide a reasonable set of capabilities and functions
-* Are the capabilities and functions useful
-* What types of applications would benefit from them
-* How would the capabilities and functions be used
-* What is the intended approach to implement them
-* A proposal is then reviewed and socialized with the wider Swift community as a “Pitch” in the Server forums, similar to how Swift Evolution handles them.
 
-### Prototyping and Development
-Once a proposal is approved by the work group and input has been received from the wider community via the [Swift Server forum](https://forums.swift.org/c/development/server), prototyping and development can begin. The work group and the greater Swift Server community will work to sponsor and promote incubation proposals, but once approved, libraries and tools can be prototyped and developed independently of the group.
+## Meeting Time
 
-Once the full set of functions has been developed, released, adopted and feedback acted on, the work group will branch a formal recommendation process.
+The SSWG meets on the 1st and 3rd Thursday of every month at 9:30AM PT (USA Pacific)
 
-### Recommendation
-As mentioned earlier, the Swift Server work group will be defining a formal process to determine how a library or tool will be recommended. Watch this space for more information as the details develop. Currently SwiftNIO is the only recommended library by the Swift Server work group.
+## Meeting Notes
 
-## Forums
-The Swift Server work group uses the [Server](https://forums.swift.org/c/development/server) forum for general discussion.
+* 
```

Comment:
Have we decided on where to post meeting notes?  We could create a sub-category of "Development > Server" for this purpose, or keep it on this repository.  We should decide.

Reply:
+1 on creating subcategories, feels like a better place for meeting notes imo. we can discuss on next meeting